tests/test_detection.py

```diff
@@ -79,3 +82,12 @@ def test_recall(reference, hypothesis):
     detectionRecall = DetectionRecall()
     recall = detectionRecall(reference, hypothesis)
     npt.assert_almost_equal(recall, 0.875, decimal=3)
+
+
+def test_fscore(reference, hypothesis):
+    # expected 28/33 since it
+    # is computed as :
+    # 2*precision*recall / (precision+recall)
+    detectionFMeasure = DetectionPrecisionRecallFMeasure()
+    precision, recall, fscore = detectionFMeasure(reference, hypothesis)
```

Comment:
For some reasons, Github Actions did not trigger tests.

Shouldn't it be like this?  

```suggestion
    fscore = detectionFMeasure(reference, hypothesis)
```
or 

```suggestion
    precision, recall, fscore = detectionFMeasure.compute_metrics(reference, hypothesis)
```